hack/tools/install_or_tools.sh

```diff
@@ -39,4 +39,5 @@ tar -zxvf ./${download_file} -C $tmp
 mv $tmp/*/bin/fzn-ortools ./bin/fzn-or-tools
 mv $tmp/*/lib*/lib*.${dyn_lib_ext}* ./lib
 chmod +x ./bin/fzn-or-tools
+cp ./bin/fzn-or-tools ./bin/solver
```

Comment:
Why do we need two copies of the solver executable?

Reply:
Leaving only one.